patches/react-native-tab-view+2.14.4.patch

```diff
@@ -0,0 +1,36 @@
+patch-package
```

Comment:
Is there a github issue open on the repo with this issue? If not, should we open one? Or use a diff lib we don't have to patch?

Reply:
Very good concern, cut a few corners to avoid spending too much time on this here.
I plan to open a PR on `react-native-tab-view`. 
The lib itself is very solid, I've used it in a bunch of past projects and a lot of effort has been put into it to provide native feeling for this.
It's was created by @satya164 which is also one of the creator of of `react-navigation`.